src/Symfony/Component/AssetMapper/ImportMap/ImportMapAuditor.php

```diff
@@ -51,14 +47,19 @@ public function audit(): array
             }
             $version = $entry->version;
 
-            $installed[$entry->importName] ??= [];
-            $installed[$entry->importName][] = $version;
+            $packageName = $entry->getPackageName();
+            $installed[$packageName] ??= [];
+            $installed[$packageName][] = $version;
 
-            $packageVersion = $entry->importName.($version ? '@'.$version : '');
-            $packageAudits[$packageVersion] ??= new ImportMapPackageAudit($entry->importName, $version);
+            $packageVersion = $packageName.'@'.$version;
```

Comment:
Bug fix - e.g. if you have `chart.js/auto`, `$packageName` will now be just `chart.js`, which is what we need for the API call. Covered with a test.

Also, `$version` is now guaranteed not null here.